src/uv_fs.c

```diff
@@ -381,7 +408,16 @@ int UvFsMakeOrOverwriteFile(const char *dir,
         goto err;
     }
 
+#ifdef WINDOWS
+    uv_buf_t convBufArr[1];
+    uv_buf_t convBuf;
```

Comment:
the separate convBuf variable is not needed, you could work with `uv_buf_t` in the array directly.

Reply:
Agree, switched to "slim" version.